### DISCUSSION TITLE
WatsonTcpServer  MessageReceived event is not triggered from MSSQL server.

### DISCUSSION THREAD

**Discussion by licentia88** [jchristn/WatsonTcp#291] [Q&A]
Hi,
I try to send data from a sql stored procedure  to the xxx:11007 port. 
I have the code from the samples where I only changed Ip and port sections.
I start my server then I trigger the stored procedure but MessageReceived never gets triggered..

it's interesting that if I change from WatsonTcpServer  to SimpleTcpServer  I can see that it hits to DataReceived event

**Comment by jchristn** [comment]
This is expected. WatsonTcp uses framing, SuperSimpleTcp does not.

**Comment by jchristn** [comment]
From the README:
![image](https://github.com/user-attachments/assets/1f7c6e65-c603-42b9-9b0e-87bdd8da89c4)
